core/src/main/java/discord4j/core/shard/GatewayBootstrap.java

```diff
@@ -194,6 +198,19 @@
         return this;
     }
 
+    /**
+     * Set the intents for the identify.
+     * For a list of linked events for each {@link Intent} check the enum inside the {@link Intent} class.
+     * Intents define which events will be send to the client.
+     *
+     * @param intents a intent-set of all intents to be subscribed
+     * @return this builder
+     */
+    public GatewayBootstrap<O> setIntents(IntentSet intents) {
```

Comment:
Presumably, intents could differ between shards. Should this have a signature closer to `setInitialPresence`?

Reply:
Do you have a citation where that's mentioned?

Reply:
Open to discussion if a use case exists for this.

Reply:
Is it even possible to log in with different intents on different shards? Anyway I myself wouldn't see any advantage in having this.